src/open_inwoner/templates/pages/profile/actions/edit.html

```diff
@@ -3,27 +3,29 @@
 
 
 {% block content %}
-<h1 class="h1">
-    {% if object.pk %}
-        {% trans "Wijzig actie" %}
-    {% else %}
-        {% trans "Creeer actie" %}
-    {% endif %}
-</h1>
+    <div class="plan-edit">
```

Comment:
Do we need the indent on the first element? It seems that we don't have a consistent policy on this. Sometimes the first element after a `block` tag is indented, sometimes it isn't.

Reply:
The indentation in the whole file looks good to me (for the block content and the div)